canonicalwebteam/blog/wordpress_api.py

```diff
@@ -18,43 +18,50 @@ def process_response(response):
     return response.json()
 
 
-def get_articles(tags, per_page=12, page=1, exclude=None, category=None):
-    url_parts = [
-        API_URL,
-        "/posts?",
-        "&per_page=",
-        str(per_page),
-        "&page=",
-        str(page),
-    ]
-
-    if tags:
-        url_parts = url_parts + ["&tags=", str(tags)]
-
-    if exclude:
-        url_parts = url_parts + ["&exclude=", str(exclude)]
-
-    if category:
-        url_parts = url_parts + ["&categories=", str(category)]
-
-    url = "".join(url_parts)
+def get_articles(
+    tags=None,
+    per_page=12,
+    page=1,
+    tags_exclude=[],
+    exclude=None,
+    categories=None,
+):
+    """
+    Get articles from Wordpress api
```

Comment:
For me, the sentence describing the function is enough. I'm not too bothered about the `:param` stuff - I think the function definition + good variable names is usually clear enough for most people. But I'm also not opposed to it, so happy to leave it until such a time as the team might pick a position.